### PR TITLE
Initial add of avaje-json-node

### PR DESCRIPTION
**Pull request by rbygrave**

### CODE REVIEW

File: json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package io.avaje.json.node.adapter;
+
+import io.avaje.json.JsonAdapter;
+import io.avaje.json.JsonReader;
+import io.avaje.json.JsonWriter;
+import io.avaje.json.node.JsonBoolean;
+
+final class BooleanAdapter implements JsonAdapter<JsonBoolean> {
```
Comment (SentryMan) [inline json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java:8]: so like why duplicate all of these adapters?

Comment (rbygrave) [reply json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java:8]: Technically they are not dups.  Boolean is not JsonBoolean etc.

Comment (rbygrave) [reply json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java:8]: I think your question is perhaps why have JsonNode at all?  ... ultimately I think that depends on the "helper" methods / functionality that JsonNode provides over "vanilla" types like List, Map, Boolean etc.

Or perhaps I misunderstand your question.

Comment (SentryMan) [reply json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java:8]: like if they do the same thing as the adapters in the main jsonb, why not just export the adaters in core to this new module

**Review by SentryMan — APPROVED** [review]
LGTM

File: json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java (hunk 1 above)
Comment (SentryMan) [reply json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java:8]: I see I have misread

**Pull request merged**